util/gc.py

```diff
@@ -37,6 +38,14 @@ def get_free():
     stats = os.statvfs(ROOT_DIR)
     return stats.f_bavail * stats.f_bsize / pow(1024,3)
 
+  def onerror(func, path, exc_info):
```

Comment:
Can you add a comment for the method definition?

Is this doing the chmod on the dir when there's an error deleting it in hopes it will work the next time?  Won't the chmod also fail if the owner is different than strider?